### DISCUSSION TITLE
Sound boost jumper configuration?

### DISCUSSION THREAD

**Discussion by tuFFo126** [UzixLS/zx-sizif-512#32] [PCB Build]
![dopalka](https://user-images.githubusercontent.com/124886433/234815755-135970d8-2ba9-452f-9c83-48a949a9198d.png)
What are the sound boost jumpers for? How should they be used?
When I wanted to save the zx-sizif configuration by closing the save jumper, after saving the configuration from the OSD menu, the options related to sound boost disappeared. I had to unplug the sound boost, restore the configuration and plug the sound boost back in, then the options related to sound boost appeared in the OSD again.

**Comment by UzixLS** [comment]
These jumpers disables SAA1099/TurboSound/GeneralSound if Sizif's magic ROM unavailable for some reason (i.e. there is addon which shadows magic rom).
Also they overrides any saved configuration even if magic ROM active.

**Reply by tuFFo126** [reply]
why did saving the configuration cause the saa/TS/GS option to disappear from the OSD menu?

**Reply by UzixLS** [reply]
Related entries isn't showing in menu only if external board isn't connected (this is determined at poweron). Probably there is bad connection between main board and extension board.

**Reply by tuFFo126** [reply]
Is something missing or a connection is not making contact? Disconnecting and shorting should help?

**Reply by UzixLS** [reply]
bad contact i suppose

**Reply by tuFFo126** [reply]
works, there must have been no contact.
